### DISCUSSION TITLE
How to force vue-tsc to ignore node_modules folder or any other folder

### DISCUSSION THREAD

**Discussion by VitalyaK** [vuejs/language-tools#3470] [General]
I tried the following and it didn't work for me.

- I tried setting the --skipLibCheck flag in the package.json file
```
{
  "scripts": {
    "type-check": "vue-tsc --noEmit --skipLibCheck true -p tsconfig.vitest.json --composite false"
  }
}
```

- I try add to tsconfig.vitest.json 
```
{
  "compilerOptions": {
    "skipLibCheck": true
  }
}
```
- And I try add exclude  to tsconfig.vitest.json 
```
{
  "exclude": ["node_modules"]
}
```

Now use "vue-tsc": "^1.8.8".

Why these options do not work? And how can I add a folder to ignore?

**Comment by rchl** [comment]
You can't ignore something that is explicitly referenced/imported somewhere. It's hard to help your case without a specific example/project.
